test/runtests.jl

```diff
@@ -47,7 +47,7 @@ end
             signalsum += abs(evt.waveforms[i].value[end])
         end
         @info signalsum
-        @test isapprox( signalsum, T(2), atol = 4e-3 )
+        @test isapprox( signalsum, T(2), atol = 0.02 )
```

Comment:
Any idea why the `signalsum` is getting worse? Is it a coarser grid now?

Reply:
I believe so. By default the initial grid is really coarse now.

Reply:
Then, I would prefer increasing the number of refinements rather than lowering the tolerance.
It might be that we miss some bugs by allowing greater tolerances.

Reply:
I agree, but the refinement will also be updated soon. 
Including keywords and default settings. 
And we might want to overhaul the tests (especially simulation of test detectors) anyhow. 
So I would prefer not to do this here.